scripts/pre_commit_linter.py

```diff
@@ -1831,8 +1712,8 @@ def all_filepaths(self):
     def _run_multiple_checks(self, *checks):
         """Run multiple checks in parallel."""
```

Comment:
I think this now needs a docstring to explain what `*checks` is (with typeinfo etc.)